src/content/en/tools/chrome-devtools/iterate/inspect-styles/css-animations.markdown

```diff
@@ -0,0 +1,177 @@
+---
+layout: shared/narrow
+title: "Inspect CSS animations"
+description: "Inspect and modify CSS animations with the Chrome DevTools 
+Animation Inspector."
+published_on: 2016-05-03
+updated_on: 2016-05-03
+order: 2
+authors:
+  - kaycebasques
+translation_priority: 0
+key-takeaways:
+  - "Record and view groups of related animations as thumbnails or
```

Comment:
s/Record/Capture/g if possible. Recording implies a start/stop flow eg. recording in timeline/network.